config/sitemap.rb

```diff
@@ -0,0 +1,58 @@
+SitemapGenerator::Sitemap.default_host = Curate.configuration.application_root_url
+
+SitemapGenerator::Sitemap.create do
+
+# config file for sitemap_generator. Generates public/sitemap.xml.gz
+# run the rake tasks to generate updated versions
+#       `rake sitemap:refresh` on production, 
+#       `rake sitemap:refresh:no_ping` on development
+#
+# https://github.com/kjvarga/sitemap_generator
+
+### static links ###
+  add '/catalog', :priority => 1, :changefreq => 'daily'
+  add '/users/sign_in', :priority => 0.8, :changefreq => 'monthly'
+  add '/contact_requests/new', :priority => 0.8, :changefreq => 'monthly'
+  add '/about_request', :priority => 1, :changefreq => 'monthly'
+  add '/terms_request', :priority => 0.8, :changefreq => 'monthly'
+  add '/coll_pol_request', :priority => 0.8, :changefreq => 'monthly'
+  add '/format_advice_requests', :priority => 0.8, :changefreq => 'monthly'
+  add '/faq_request', :priority => 0.8, :changefreq => 'monthly'
+  add '/creators_rights_request', :priority => 0.8, :changefreq => 'monthly'
+
+### major facets ###
+  add '/catalog?f[human_readable_type_sim][]=Article', :priority => 0.8, :changefreq => 'monthly'
+  add '/catalog?f[human_readable_type_sim][]=Document', :priority => 0.8, :changefreq => 'monthly'
+  add '/catalog?f[human_readable_type_sim][]=Dataset', :priority => 0.8, :changefreq => 'monthly'
+  add '/catalog?f[human_readable_type_sim][]=Image', :priority => 0.8, :changefreq => 'monthly'
+  add '/catalog?f[human_readable_type_sim][]=Video', :priority => 0.8, :changefreq => 'monthly'
+  add '/catalog?f[human_readable_type_sim][]=Collection', :priority => 0.8, :changefreq => 'monthly'
+  add '/catalog?f[human_readable_type_sim][]=Generic+Work', :priority => 0.8, :changefreq => 'monthly'
+  add '/catalog?f[human_readable_type_sim][]=Person', :priority => 0.8, :changefreq => 'monthly'
+
+
+### curation concerns ###
+
+  Article.find_each do |article|
```

Comment:
There's an opportunity for something a little easier to maintain: `Curate.configuration.registered_curation_concern_types` should loop through the work types, preventing us from having to keep the sitemap up to date as we add additional work types.